src/client/src/rt-components/popup/styled.tsx

```diff
@@ -0,0 +1,29 @@
+import { darken } from 'polished'
+import { styled } from 'rt-theme'
+
+export const PopupContainer = styled.div<{ open: boolean }>`
+  display: ${({ open }) => (open ? 'flex' : 'none')};
+  position: absolute;
+  z-index: 5;
+  align-items: center;
+  justify-content: center;
+  font-size: 0.75rem;
+  border-radius: 0.25rem;
+  background: ${({ theme }) => theme.core.alternateBackground};
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05),
```

Comment:
Be consistent with `rem`

Reply:
Copied it from Modal. I can change it though, and change it in Modal too? (How do I test modal? Where is it used?)

Reply:
Changed it in Popup (also enabled it for the status button popup, with Carlos' agreement)